Bio/Phylo/PAML/_parse_codeml.py

```diff
@@ -411,7 +412,7 @@ def parse_pairwise(lines, results):
     #
     # t= 0.0126  S=    81.4  N=   140.6  dN/dS= 0.0010  dN= 0.0000  dS= 0.0115
     pair_re = re.compile("\d+ \((.+)\) ... \d+ \((.+)\)")
-    pairwise = {}
+    pairwise, seq1, seq2 = collections.defaultdict(dict), None, None
```

Comment:
As noted by @brandoninvergo using three separate lines would be more in line with the rest of Biopython here.